src/cow-react/modules/swap/containers/NewSwapWidget/propsChecker.ts

```diff
@@ -60,5 +60,24 @@ export function swapPagePropsChecker(prev: SwapFormProps, next: SwapFormProps):
 }
 
 export function genericPropsChecker(prev: any, next: any): boolean {
```

Comment:
this feels like a pretty generic code. Should we move it to an util? 

Also, I think it would be nice some comments in jsdoc for what the function is for.

Reply:
If this is some kind of "deepEquals" shouldn't we use some batte tested lib?

Reply:
Thanks!
Done

Reply:
Okay! I didn't make it a blocking comment. Gave the approval in case this is adding some overhead now to find something suitable